sprocs/course_instance_permissions_update_role.sql

```diff
@@ -0,0 +1,55 @@
+CREATE OR REPLACE FUNCTION
+    course_instance_permissions_update_role(
```

Comment:
Marking this as resolved, since both of these sprocs now serve different use-cases: `_insert` can insert a new permission or update it to a higher permission, but never to a lower permission. `_update_role` can increase or decrease the permission.